From 00e8526246ef292a5071303f25877fcf8c0a2dc8 Mon Sep 17 00:00:00 2001
From: st4lk <myhappydo@gmail.com>
Date: Mon, 26 Jan 2015 10:07:12 +0300
Subject: [PATCH] apply fix #134 for babel issue #133

---
 babel/dates.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/babel/dates.py b/babel/dates.py
index 73d54fa..a1b4cd7 100644
--- a/babel/dates.py
+++ b/babel/dates.py
@@ -308,7 +308,9 @@ def get_timezone_gmt(datetime=None, width='long', locale=LC_TIME):
         datetime = datetime.replace(tzinfo=UTC)
     locale = Locale.parse(locale)
 
-    offset = datetime.tzinfo.utcoffset(datetime)
+    # fix for mitsuhiko/babel#133 : Any time will do since we just want the
+    # offset
+    offset = datetime.tzinfo.utcoffset(datetime_.utcnow())
     seconds = offset.days * 24 * 60 * 60 + offset.seconds
     hours, seconds = divmod(seconds, 3600)
     if width == 'short':
